merchant-app/app/build.gradle

```diff
@@ -0,0 +1,55 @@
+apply plugin: 'com.android.application'
+
+android {
+    compileSdkVersion 29
+    buildToolsVersion "29.0.3"
+
+    defaultConfig {
+        applicationId "com.example.smb"
+        minSdkVersion 21
+        targetSdkVersion 29
+        versionCode 1
+        versionName "1.0"
+
+        testInstrumentationRunner "androidx.test.runner.AndroidJUnitRunner"
+        multiDexEnabled true
+    }
+
+    buildTypes {
+        release {
+            minifyEnabled false
+            proguardFiles getDefaultProguardFile('proguard-android-optimize.txt'), 'proguard-rules.pro'
+        }
+    }
+    aaptOptions {
+        noCompress "tflite"
+    }
+    packagingOptions {
+        exclude 'META-INF/androidx.exifinterface_exifinterface.version'
+        exclude 'META-INF/proguard/androidx-annotations.pro'
+    }
+}
+
+dependencies {
+    implementation fileTree(dir: 'libs', include: ['*.jar'])
+
+    implementation 'androidx.camera:camera-camera2:1.0.0-alpha04'
+    implementation 'androidx.camera:camera-core:1.0.0-alpha04'
+    implementation 'androidx.appcompat:appcompat:1.1.0'
+    implementation 'androidx.constraintlayout:constraintlayout:1.1.3'
+    implementation 'androidx.preference:preference:1.1.1'
+    implementation 'androidx.media:media:1.1.0'
+    implementation 'com.google.android.material:material:1.1.0'
+    // Firebase analytics
+    implementation 'com.google.firebase:firebase-analytics:17.2.2'
+    // ML Kit dependencies
+    implementation 'com.google.firebase:firebase-ml-vision:24.0.3'
+    // Barcode detection model.
```

Comment:
Use consistent comment convention. Can you remove the full stop at the end.

Reply:
done